lib/sso_openid/urls.yml

```diff
@@ -21,13 +24,19 @@ defaults:
     dashboard: "https://account-uat.networkforgood.org/Npo/Dashboard.aspx"
     disbursements: "https://account-uat.networkforgood.org/Npo/AccountManagement/OrganizationInfo.aspx"
 test:
+  auctions:
+    host: "n/a"
+    fqdn: "n/a"
   dms:
     host: "dm.n4g.be:3001"
     fqdn: "http://dev.dm.n4g.be:3001"
   evo:
     host: "n4g.be:3000"
     fqdn: "http://basic.n4g.be:3000"
 development:
+  auctions:
+    host: "auctions.test"
+    fqdn: "http://dev.auctions.test"
```

Comment:
is it an entity with a subdomain of "dev" that the seeds create? Feels like it should be "basic" as that is what is created in the FP seeds, and as we discussed yesterday, the subdomains of matching entities in FP and Auctions have to match

Reply:
I see a `dev`, `basic`,`admin` Entities in the seed file.

Reply:
Based on our conversation dev is ok.